cfgs/circusc.cfg

```diff
@@ -0,0 +1,4 @@
+input_overlay = "/opt/retropie/emulators/retroarch/overlays/arcade-bezels/circusc.cfg"
+input_overlay_enable = "true"
+input_overlay_opacity = "0.900000"
```

Comment:
Please put this as 1.000000

Reply:
Damn, I thought I had this fixed in my template already. Done!